packages/ui/src/utils/button.ts

```diff
@@ -0,0 +1,84 @@
+import { css, DefaultTheme } from 'styled-components';
+
+export type ActionType = 'default' | 'accent' | 'unshield' | 'destructive';
+
+export type Priority = 'primary' | 'secondary';
```

Comment:
question: why is this called `priority`? And why is it controlled by `ButtonPriorityContext` and not just set by user?

Reply:
I need to write some docs for some of the choices Sam and I are making regarding naming in the UI library; one of them is that `priority` represents the choice of a `primary` vs `secondary` item in a set (rather than using a generic term like `variant`). The idea is that our props should indicate to devs how the components are _used_, rather than how they _look_.

Re: the context, see [here](https://github.com/penumbra-zone/web/pull/1516/files#diff-d116d5fdfb5c55efb62165b5508b1e0edb0f3cb3c0204ad804215e9295b815ba).